actions/docs-verifier/src/RedirectionVerifier/NavigationOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+using System.Text.Json.Serialization;
+
+namespace RedirectionVerifier
+{
+    public record NavigationOptions(
+        [property: JsonPropertyName("maximumNumberOfArticles")] int MaximumNumberOfArticles,
+        [property: JsonPropertyName("tocParentNode")] string? TocParentNode,
+        [property: JsonPropertyName("repoTocFolder")] string? RepoTocFolder,
+        [property: JsonPropertyName("indexParentNode")] string? IndexParentNode,
+        [property: JsonPropertyName("repoIndexFolder")] string? RepoIndexFolder)
```

Comment:
Can the unused properties by omitted? or they're required by `System.Text.Json`?

Reply:
They could be omitted, but I left them there for now.